src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -63,7 +63,6 @@ public IVoidConfiguration WhenArgumentsMatch(Func<ArgumentCollection, bool> argu
 
         public virtual IAfterCallSpecifiedConfiguration DoesNothing()
         {
-            this.RuleBeingBuilt.Applicator = x => { };

```

Comment:
> @thomaslevesque, in this example, DoesNothing will prevent the call to base.SomeMethod(). It works because the repeated CallTo creates a new rule on which the applicator lives. The DoesNothing will leave the default applicator on the rule, and then it's used when the call is made.

Good point. I'm just back from holiday and my brained isn't fully turned back on yet :wink: